tests/e2e/seed/scripts/deploy.ts

```diff
@@ -24,43 +28,65 @@ import { JsonRpcProvider } from "@ethersproject/providers";
 const tokenName = "Art Blocks V3 Core Dev";
 const tokenTicker = "BLOCKS_V3_CORE_DEV";
 const superAdminAddress = undefined; // set to undefined to use deployer address
+const minterFilterSuperAdminAddress = undefined; // set to undefined to use deployer address
 const artblocksPrimarySalesAddress = undefined; // set to undefined to use deployer address
 const artblocksSecondarySalesAddress = undefined; // set to undefined to use deployer address
 const startingProjectId = 0; // offset from existing core with margin for new projects in the next ~month
-// (optional) add initial project
-const doAddInitialProject = false;
-const initialProjectName = undefined;
-const initialProjectArtistAddress = undefined;
 //////////////////////////////////////////////////////////////////////////////
 // CONFIG ENDS HERE
 //////////////////////////////////////////////////////////////////////////////
 
+type ConfigMetadata = {
+  [key: string]: { address: string }[] | string;
+};
+
 type SubgraphConfig = {
+  [key: string]: { address: string }[] | string | ConfigMetadata | undefined;
   network: string;
-  [key: string]: { address: string }[] | string;
+  metadata?: ConfigMetadata;
 };
 
 async function main() {
   const subgraphConfig: SubgraphConfig = { network: "mainnet" };
+  subgraphConfig.metadata = {};
 
+  /////////////////////////////////////////////////////////////////////////////
+  // SETUP ACCOUNTS BEGINS HERE
+  /////////////////////////////////////////////////////////////////////////////
   const accounts = JSON.parse(
     fs.readFileSync("./shared/accounts.json", "utf8")
   );
   const deployer = ethers.Wallet.fromMnemonic(accounts.mnemonic).connect(
     new JsonRpcProvider("http://hardhat:8545")
   );
+  const artist = ethers.Wallet.fromMnemonic(
+    accounts.mnemonic,
+    "m/44'/60'/1'/0/1" // derivation path index 1
```

Comment:
Just curious, what is this exactly?

Reply:
It is the bip-44 derivation path for account 1.
I slightly messed this up and should have used `address_index=0`, so I've updated and added a little more context:
```
"m/44'/60'/1'/0/0" // bip-44 derivation path Ethereum account 1`
```
The intent here is just to get the "next" account from the seed phrase that was stored during the deployment process, and assign it to artist.
refs:
- https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels
- https://docs.ethers.org/v5/api/signer/#Wallet.fromMnemonic